docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+@page guides/contributing/include-new-modules Including New Modules
+@parent guides/contribute
+
+@description Learn how to add your module to this site.
+
+@body
+
+A project that is useful to the CanJS community can be included on the [CanJS documentation](http://canjs.com/) site.
+
+Before submitting a PR with these changes, make sure that the source module has [guides/contributing/documentation] and tests that follow [guides/contributing/code] standards set forth by the CanJS community.
+
+## Source Repo Documentation
+
+Create a markdown file in the `docs` folder of source repo and name it after the project.
+
+```bash
+# This example puts the documents in the project root
+cd my-fancy-utility
+touch my-fancy-utility.md
+```
+
+## Source Repo Documentation Tags
+
+In order to include the source project's documentation on the CanJS site, the main document file requires some tags.
+
+```javadocs
+@ module {Object} <MODULE_NAME>
+@ parent <CANJS_PARENT_TARGET>
+@ package <PATH_TO_PACKAGE.JSON>
+```
+
+- `MODULE_NAME`: The project name, which should match the `name` property in the source project's `package.json`.
```

Comment:
You can make this consistent with the other list items by replacing `: The project` with `: is the project`